src/main/java/com/zipcodewilmington/phone/PhoneNumberFactory.java

```diff
@@ -38,15 +61,24 @@ private static PhoneNumber createRandomPhoneNumber() {
      * @return a new phone number object
      */ //TODO - if input is valid, return respective PhoneNumber object, else return null
     public static PhoneNumber createPhoneNumberSafely(int areaCode, int centralOfficeCode, int phoneLineCode) {
-        return createPhoneNumber(null);
+        try {
+            String phoneNumberString = "(" + areaCode + ")-" + centralOfficeCode + "-" + phoneLineCode;
+            createPhoneNumberSafelyLogger.log(Level.WARNING, phoneNumberString+ " is not a valid number.");
+            return createPhoneNumber(phoneNumberString);
+        }catch (InvalidPhoneNumberFormatException ipnfe){
+            return null;
+        }
     }
 
     /**
      * @param phoneNumberString - some String corresponding to a phone number whose format is `(###)-###-####`
      * @return a new phone number object
      * @throws InvalidPhoneNumberFormatException - thrown if phoneNumberString does not match acceptable format
      */ // TODO - Add throws statement to method signature
-    public static PhoneNumber createPhoneNumber(String phoneNumberString) {
-        return null;
+    public static PhoneNumber createPhoneNumber(String phoneNumberString) throws InvalidPhoneNumberFormatException{
+        createPhoneNumberLogger.log(Level.WARNING, "Attempting to create a new PhoneNumber object with a value of "+phoneNumberString);
```

Comment:
`logger.log(Level.INFO, "Attempting to create a new PhoneNumber object with a value of "+phoneNumberString)`